File: 1-codes/micro/micro02.c
#import<stdio.h>

int main(){
  int num1, num2;
  printf("Digite o primeiro numero: ");
  scanf("%d", &num1);
  printf("Digite o segundo numero: ");
  scanf("%d", &num2);

  if(num1 > num2) {
    printf("O primeiro numero %d e maior que o segundo %d.", num1, num2);
  } else {
    printf("O primeiro numero %d e maior que o segundo %d.", num2, num1);
  }

	return 0;
}

File: 3-sintatico/tests/micro02.py
def main() -> None:
    num1 = 0
    num2 = 0
    print("Digite o primeiro numero: ")
    int(input(num1))
    print("Digite o segundo numero: ")
    int(input(num2))

    if num1 > num2:
        print("\nO primeiro numero ")
        print(num1)
        print(" e maior que o segundo ")
        print(num2)
    else:
        print("\nO segundo numero ")
        print(num2)
        print(" e maior que o primeiro ")
        print(num1)

main()
